### PR TITLE
feat: modify refs_commits_diffs table name and struct

### PR DESCRIPTION
**Pull request by abeizn**
### ⚠️ Pre Checklist

> Please complete _ALL_ items in this checklist, and remove before submitting

- [x] I have read through the [Contributing Documentation](https://devlake.apache.org/community/).
- [x] I have added relevant tests.
- [x] I have added relevant documentation.
- [ ] I will add labels to the PR, such as `pr-type/bug-fix`, `pr-type/feature-development`, etc.

# Summary
In order to increase the scalability of commits diffs, now the refs_commits_diffs table needs to modify.
1. rename refs_commits_diffs to commits_diffs.
2. upgrade the  commits_diffs struct.
3. add commits_diffs_status table.

### Does this close any open issues?
related to #3685 

### Screenshots
The new tables show:
image

copy data result:
pre:
image

now:
image

### Other Information
This is just a migration script, the model will be processed in the next pr

### CODE REVIEW

File: models/migrationscripts/20221109_modfiy_commits_diffs.go [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+	"github.com/apache/incubator-devlake/helpers/migrationhelper"
+	"github.com/apache/incubator-devlake/plugins/core"
+)
+
+var _ core.MigrationScript = (*modifyCommitsDiffs)(nil)
+
+type modifyCommitsDiffs struct{}
+
+// initial format
+type RefsCommitsDiff20221109 struct {
```
Comment (klesh) [inline models/migrationscripts/20221109_modfiy_commits_diffs.go:31]: What does it do? I don't see any references to this struct.

File: models/migrationscripts/20221109_modfiy_commits_diffs.go [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+}
+
+func (RefsCommitsDiff20221109) TableName() string {
+	return "refs_commits_diffs"
+}
+
+// updated format
+
+type CommitsStatus20221109 struct {
+	NewRefCommitSha string `gorm:"primaryKey;type:varchar(40)"`
```
Comment (klesh) [inline models/migrationscripts/20221109_modfiy_commits_diffs.go:47]: `Ref` represent `tag`/`branch`, not needed anymore.
The same apply to the rest of the source code

File: models/migrationscripts/20221109_modfiy_commits_diffs.go (hunk 1 above)
Comment (abeizn) [reply models/migrationscripts/20221109_modfiy_commits_diffs.go:31]: This is what I added on purpose, in order to see the structure of the previous table

File: models/migrationscripts/20221109_modfiy_commits_diffs.go (hunk 2 above)
Comment (abeizn) [reply models/migrationscripts/20221109_modfiy_commits_diffs.go:47]: i forgot the name, i will update `NewRefCommitSha` to `NewCommitSha`, same to OldRefCommitSha

**Review by mindlesscloud — APPROVED** [review]
LGTM

**Pull request merged**